### PR TITLE
Clean up the table: Rework and balance of "Set the table and clean up"

### PR DESCRIPTION
**Pull request by kyordhel**
The test was simplified focusing mostly in the cleaning part. The robot no longer serves any meal (so far untried), and focuses in manipulation of cutlery, tableware, and spot removal.

Changes:
- Renamed to *Clean up the table*
  - Internal files renamed
- Setup:
  - Test takes place in the arena (kitchen).
  - Table is already set, robot cleans.
  - There is only a spot (high contrast).
- Task:
  - Standard test start: door
  - All cutlery and tableware is moved to the sink.
  - Alternative use of dishwasher (prep for dishwasher test)
  - Object categorization: food leftovers are either:
    - Disposed
    - Stored for later use
  - Robot shall leave (bonus points for leaving)
- Scoresheet
  - Reworked
  - Simplified scoring

### CODE REVIEW

File: tests/CleanTheTable.tex [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+	\item \textbf{Objects:} A total of 6 objects are used in this test following the distribution shown below:
+	\begin{itemize}
+		\item\textit{Silverware}: Any two objects.
+		\item\textit{Tableware}: Any two objects, excluding silverware. At least one must be a dish or a bowl.
+		\item\textit{Food leftovers}: Any two edible or drinkable objects, regardless its category. One shall be disposed and the other stored for later use.
+	\end{itemize}
+	All objects used in this test are taken from the list of standard objects (See \ref{rule:scenario_objects}). Tableware and silverware are considered to be known to the robot.
+
+	\item \textbf{Spots and Spills:} The referees must place a spot (e.g. jam or chocolate syrup) or spill some liquid (e.g.milk) on the table before the test stats. The substance used to create the spot shall be clearly visible and contrasting with the table. When cleaning it, it must be clear the robot has detected the spot and is trying actively to clean it. The selection of the cleaning tool (sponge or cloth) is made by the team.\\
```
Comment (LoyVanBeek) [inline tests/CleanTheTable.tex]: "Before the test stats" --> "starts"

File: tests/CleanTheTable.tex [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+   \item Images of recognized objects
+   \item List of moved items
+  \end{itemize}
+
+\subsection{Referee instructions}
+
+The referee needs to
+\begin{itemize}
+	\item Place the table setting.
+	\item Clean spots smudged by the robot.
```
Comment (LoyVanBeek) [inline tests/CleanTheTable.tex]: By the previous robot, that is?

File: scoresheets/CleanTheTable.tex [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
+The maximum time for this test is \textbf{10 minutes}.
+
+\begin{scorelist}
+
+	\scoreheading{Clearing the table} %190 pts
+	\scoreitem{20}{Successfully dumping disposable food leftovers in the bin}
+	\scoreitem{20}{Successfully placing other edibles in their predefined location}
+	\scoreitem[2]{35}{Safely placing a tableware item in the sink}
```
Comment (LoyVanBeek) [inline scoresheets/CleanTheTable.tex]: sink OR dishwasher

File: scoresheets/CleanTheTable.tex [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	\scoreitem[2]{40}{Safely placing a cutlery item in the sink}
```
Comment (LoyVanBeek) [inline scoresheets/CleanTheTable.tex]: sink OR dishwasher

File: tests/CleanTheTable.tex (hunk 1 above)
Comment (kyordhel) [reply tests/CleanTheTable.tex]: Thank you for this one.

File: tests/CleanTheTable.tex (hunk 2 above)
Comment (kyordhel) [reply tests/CleanTheTable.tex]: Yup, that is.

File: scoresheets/CleanTheTable.tex (hunk 3 above)
Comment (kyordhel) [reply scoresheets/CleanTheTable.tex]: Thanks.

File: scoresheets/CleanTheTable.tex (hunk 4 above)
Comment (kyordhel) [reply scoresheets/CleanTheTable.tex]: Another one.

**Pull request closed without merging**